action/receipt.go

```diff
@@ -40,6 +40,7 @@ type (
 		logs               []*Log
 		transactionLogs    []*TransactionLog
 		executionRevertMsg string
+		LogIndex           uint32 // the current log index, NOT part of protobuf
```

Comment:
One receipt might include a number of logs(like 0,1,2)

Reply:
yes, in that case, receipt.LogIndex = 3 (meaning 3 logs are generated inside EVM)
and in workingset.go,
```
logIndex = receipt.LogIndex
// next loop
ctx, err = withActionCtx(ctx, nextAction, txIndex, logIndex)
```
so the next action will start with index = 3

Reply:
`NextLogIndex` would be more easy to read

Reply:
one cleaner way is to use func() read last #log in the `logs` filed
Pseudocode would like below
```
func (re *receipt) NextLogIndex() uint32 {
   return  re.logs[len(re.logs)-1].Index
}
```

Reply:
good catch, then don't need to add this field into receipt